main.py

```diff
@@ -1,12 +1,226 @@
+            print(f"The customer you selected is: {selected_customer}")
+            
+    elif choice == "4":
+    # Create a customer
+        print("Okay, I'm ready to create a customer.")
+        name=input('What is their name? ')
+        postal_code=input('What is their postal code? ')
+        phone=input('What is their phone number? ')
+        response = customer_list.create_customer(name=name, postal_code=postal_code,phone=phone)
+
+        print(f"You created a new customer with the id of: {response['id']}")
+        
+    elif choice == "5":
+    # Update a customer
+        print("Lets update that customer's info!")
+        # get id
+        id = input('What is their id? ')
+        id = int(id)
+        # select customer
+        selected_customer = customer_list.get_customer(id=id)
+        print(f"The customer you selected is: {selected_customer}")
+        # get attributes
+        name=input("What is the new name for your customer?")
+        postal_code=input("What is the new postal code for your customer?")
+        phone=input("What is the new phone number for your customer?")
+        # create response
+        response = customer_list.update_customer(name=name,postal_code=postal_code,phone=phone)
+        # print output
+        print_decoration()
+        print(f"You updated that customer!")
+        print(response)
+        print_decoration()
+
+    elif choice == "6":
+    # Delete a customer
+        print("Lets put that customer in the garbage!")
+        id = input('What is their id? ')
+        id = int(id)
+        selected_customer = customer_list.get_customer(id=id)
+        print(f"The customer you selected is: {selected_customer}")
+        # select customer
+        response = customer_list.delete_customer(id=id)
+        print(response)
+        print("You deleted that customer!")
+            
+    elif choice == "7":
+        # get a video
+        print("Okay, I'm ready to get a video.")
+        id = input('What is the id of the video? ')
+            # check if digit. 
+        if id.isnumeric():
+            id = int(id)
+            selected_video = video_list.get_video(id=id)
+            print(f"The video you selected is: {selected_video}")
+    elif choice == "8":
+        # create a video
+        print("Okay, I'm ready to create a video.")
+        title=input('What is the title? ')
+        release_date=input('What is the release date? ')
+        total_inventory=int(input('What is the total inventory? '))
+        response = video_list.create_video(title=title, release_date=release_date,total_inventory=total_inventory)
+        print(f"You created a new video with the id of: {response['id']}")
+    
+    elif choice == "9":
+        # update a video
+        print("Lets update that video's info!")
+        # get id
+        id = input('What is the id? ')
+        id = int(id)
+        # select video
+        selected_video = video_list.get_video(id=id)
```

Comment:
Currently, I can enter a video id that doesn't exist and continue the flow of your CLI. As a refactor, it'd be good to add some logic to prevent updates to videos that don't exist.